srcpkgs/audacious-plugins/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'audacious-plugins'
 #Keep in sync with audacious!
 pkgname=audacious-plugins
-version=4.0.5
+version=4.1
```

Comment:
GitHub won't let me add a suggestion for unchanged lines, but since 4.1 now includes the OpenMPT plugin I would greatly appreciate if `libopenmpt-devel` was added to `makedepends`.

Reply:
Thanks, done!

Reply:
Shouldn't `libopenmpt-devel` have gone in `audacious-plugins`? It doesn't seem to be used by `audacious` itself.

Reply:
And configure shows:

```
2021-02-07T20:50:22.5217415Z checking for OPENMPT... no
2021-02-07T20:50:22.5320376Z configure: WARNING: libopenmpt disabled due to missing dependency: libopenmpt >= 0.2
```

Reply:
Yes, ofc... :)